lib/type/unit/function/createUnit.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var deepMap = require('../../../utils/collection/deepMap');
+
+function factory (type, config, load, typed) {
+}
+
+exports.name = 'createUnit';
+exports.factory = factory;
```

Comment:
Is this necessary for the tests to work? If not, please remove, and also make sure to pull those changes into the debug PR

Reply:
Also please make sure that after any changes you try running the tests on both the test and debug prs!

Reply:
> Is this necessary for the tests to work? If not, please remove, and also make sure to pull those changes into the debug PR

It is important for the tests to work, because otherwise tests will not work.

What I did here is typed a function builder plate that will be implemented in debug PR